src/main/java/com/google/sps/servlets/AutoEventProcessor.java

```diff
@@ -0,0 +1,318 @@
+
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.api.client.auth.oauth2.BearerToken;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.auth.oauth2.TokenResponse;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.Calendar.Freebusy;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+import com.google.sps.util.DatastoreModule;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import com.google.api.services.calendar.model.FreeBusyRequest;
+import com.google.api.services.calendar.model.FreeBusyRequestItem;
+import com.google.api.services.calendar.model.FreeBusyResponse;
+
+@WebServlet("/process-user-data")
+public class AutoEventProcessor extends HttpServlet {
+
+    private static final long serialVersionUID = 1L;
+    private static final String APPLICATION_NAME = "Meeting Manager";
+    final private static JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+    public void init() {
+        try {
+            DatastoreModule.init();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static String stripQuotes(String input) {
+        return input.replaceAll("\"", "");
+    }
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse res) {
+    }
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOException {
+
+        res.setContentType("application/json;");
+
+        // Users get sent as a JsonArray of Strings, use Gson Library to load them into
+        // a string array
+        String[] users = new Gson().fromJson(req.getParameter("userList"), String[].class);
+
+        /* Scheduling Constraints */
+        int startConstraint = Integer.parseInt(req.getParameter("startRes"));
+        int endConstraint = Integer.parseInt(req.getParameter("endRes"));
+        int meetingLength = Integer.parseInt(req.getParameter("meetingLength"));
+        
+        try {
+
+            List<Calendar> calendars = getCalendars(users);
+
+            int[][] freeBusyTimes = getFreeBusyTimes(calendars);
+            /* 
+                 Below is for Testing
```

Comment:
I would recommend keeping commented out code out of committed code. It adds unnecessary lines.